tests/std/tests/strict_winsdk_concepts_matrix.lst

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# This file is identical to concepts_matrix.lst, except that:

```

Comment:
Ditto :rotating_light: Stealth Merge Conflict :rotating_light:.

Reply:
@CaseyCarter took care of it